<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UtilTest</class>
 <widget class="QWidget" name="UtilTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>144</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="directoryLabel">
     <property name="text">
      <string>Choose Directory:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ctkDirectoryButton" name="directoryButton">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="checkBoxLabel">
     <property name="text">
      <string>Create label map segmentations</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ctkCheckBox" name="labelmapCheckbox"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ctkCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkDirectoryButton</class>
   <extends>QWidget</extends>
   <header>ctkDirectoryButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
